app/browserPage/index.tsx

```diff
@@ -62,25 +52,16 @@ export default function BrowserPage() {
           </DefaultView>
         </DefaultView>
 
-        <DefaultView style={{ marginLeft: width * 0.55 }}>
-          <CryptoDropdown onPress={onPress} text="Polygon" logo={<Polygon />} />
-        </DefaultView>
+        <CryptoDropdown onPress={onPress} text="Polygon" logo={<Polygon />} />
       </DefaultView>
       <WebView
         source={{ uri: url as string }}
         allowFileAccess
         scalesPageToFit
         originWhitelist={["*"]}
+        setSupportMultipleWindows={false}
       />
-      <SwipeModal />
-    </DefaultView>
+      {Platform.OS === "ios" && <SwipeModal />}
```

Comment:
Is the modal not applicable to `android`?

Reply:
The modal on the wallet screen is not removed on android, making it rendered twice.